### DISCUSSION TITLE
We need to add BUS

### DISCUSSION THREAD

**Discussion by UkrainianBanderasCat** [UkrainianBanderasCat/Digital-Logic-Sim2#8] [General]
Does anyone have any ideas on how to make a Bus?

And we need to add to this Bus able to stick, so people wouldn't connect every pin to chip manually.

**Comment by firecerne** [comment]
Hello , could you explain what is a Bus a what do you mean by stick ?

**Reply by UkrainianBanderasCat** [reply]
Bus it's many connections are in one and stick that people don't need to drag every wire to Pin it's doing automatically

**Reply by hodytone** [reply]
It shoul be different pin type - "buspins" with width property so you can connect only chips with the same width bus. Also we need bus2pin converters to connect several pins to bus and back.

**Comment by RobMayer** [comment]
https://github.com/SebLague/Digital-Logic-Sim/pull/47

**Reply by UkrainianBanderasCat** [reply]
So... Can I use your scripts to merge with mine?

**Comment by RobMayer** [comment]
it's not mine, unfortunately. I just realized that there was another fork
of the project that might be worth collaborating with.
